src/Compilers/CSharp/Portable/SourceGeneration/CSharpGeneratorDriver.cs

```diff
@@ -76,6 +76,15 @@ internal override SyntaxTree ParseGeneratedSourceText(GeneratedSourceText input,
 
         internal override string SourceExtension => ".cs";
 
+        internal override string EmbeddedAttributeDefinition => """
+            namespace Microsoft.CodeAnalysis
```

Comment:
Nit.  File scoped namespace

Reply:
I intentionally did not do this to maximize language version compat.